config/test.js

```diff
@@ -61,4 +61,45 @@ module.exports = {
       "Zzn584h/xbSIcP9/4SXmEK3wtoueUtMkvS3yKTvk+w==\n" +
       "-----END RSA PRIVATE KEY-----",
   },
+
+  botToken: {
+    botPublicKey:
+      "-----BEGIN PUBLIC KEY-----\n" +
+      "MIIBITANBgkqhkiG9w0BAQEFAAOCAQ4AMIIBCQKCAQBK3CkprcpAYxme7vtdjpWO\n" +
+      "gFFjoYsqU3OmhMEty/s1gnW5tgbK4ief4xk+cU+mu3YvjzWudT/SV17tAWxL4Y+G\n" +
+      "incJwL5gpQwlnw9qOAdRGkpBriQLec7kNVIydZXbUitziy+iSimxNzdDmjvlK9ZG\n" +
+      "miVLZm+MePbUtgaIpfgd+4bRWzudlITiNmWY7HppLzyBw+037iEICM4kwPPFI+SO\n" +
+      "GJhpAAmD6vk0MeZk1NeQmyQp/uOPpWmVRzgyK+XVc6AwZHV+/n6xAIT91/DjJlD1\n" +
+      "N+nS7Sqo3RJ04+KlNRUclzINOC7JBYkKtG7YQ0U9nNLkRrRlON+O6tY4OT86T1O1\n" +
+      "AgMBAAE=\n" +
+      "-----END PUBLIC KEY-----",
+    botPrivateKey:
```

Comment:
Why are we storing confidential information

Reply:
This is only for testing purposes and without the dummy keys, we won't be able to test the API. The same is being done for `userToken` also.

Reply:
But Isn't this some account data that is registered, by providing this public.... Does it lead to anyone accessing ?

Reply:
No, it's not registered with any account and do not lead any data access. I have created it for testing purposes from openssl.

Reply:
Got it, then great :D

Reply:
@harshith-venkatesh the original application is using the keys from github secrets. So rest assured, we are using a fresh set of keys for prod and staging.